Automattic/simplenote-ios
# Update Dark Palette to IOS Standards [WIP]

### Fix
iOS dark colours have been pulled directly from Android. This is a PR to replace the Android palette with an iOS standard dark theme

Before             |  After
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/36532468/74673805-edd88600-51a7-11ea-9525-f01959e53b2d.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74721195-eb6d3f00-522e-11ea-962d-51a90ef720a9.PNG)
![](https://user-images.githubusercontent.com/36532468/74673926-24160580-51a8-11ea-8544-0b628b2e3888.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74722241-80bd0300-5230-11ea-8c42-3dde99b4dd75.PNG)
![](https://user-images.githubusercontent.com/36532468/74721144-d2fd2480-522e-11ea-903d-04ff40562908.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/74721076-bc56cd80-522e-11ea-99bc-5e0a558db580.PNG)

### Test
1. Switch app to dark theme
2. Notice dark theme has been updated to iOS standard dark theme

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
